docs/developer/release/release_push.py

```diff
@@ -149,11 +150,11 @@ def check_all_links(afu_website, checker_website, suffix, test_mode, checker_ver
         if not test_mode:
             release_option = " release"
         raise Exception("The link checker reported errors.  Please fix them by committing changes to the mainline\n" +
-                        "repository and pushing them to GitHub/Bitbucket, running \"python3 release_build.py all\" again\n" +
-                        "(in order to update the development site), and running \"python3 release_push" + release_option + "\" again.")
+                        "repository and pushing them to GitHub, running \"python release_build.py all\" again\n" +
```

Comment:
I think this should be `python3` instead of `python` which might be Python 2 or Python 3 depending on the system.

Reply:
These scripts are not yet written in python3.

Reply:
The [shebang line](https://github.com/typetools/checker-framework/blob/master/docs/developer/release/release_build.py#L1) contains `python3`.
If that is incorrect, the shebang line should be fixed.

Reply:
I thought these scripts needed to be converted to Python 3 and that you haven't done that yet.  I see that you changed the shebang on Dec. 21 afd6923c, did you also convert the scripts?

Reply:
Good point -- just changing the shebang line is not enough.  I have opened https://github.com/typetools/checker-framework/pull/4134.